src/content/tutorials/connectivity/ingress/configuration/index.md

```diff
@@ -0,0 +1,640 @@
+
+Don't copy all the defaults if you don't need to change them, that way we can adjust them in case they need to change.
+
+Do make sure you look at the right tag of that repository, when reading this file check that the tag corresponds to the version of the ingress nginx controller running on your cluster.
+
+### Configure Proxy Protocol
+
+__Warning__:
+
+We also allow setting `use-proxy-protocol: "true"/"false"`. This setting always applies globally for the ingress nginx controller.
+
+#### AWS / CAPA
+
+The proxy protocol is enabled by default. It can be disabled by setting the `use-proxy-protocol` to `"false"`. For example:
+
+```yaml
+# On the management cluster
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ingress-nginx-user-values
+  namespace: NAMESPACE
+data:
+  values: |
+    controller:
+      config:
+        use-proxy-protocol: "false"
+```
+
+---
+
+### Default certificate
+
+When you want to have the default server on the ingress nginx controller support TLS you need to provide a certificate. This is configured using the flag `--default-ssl-certificate`. Now you can provide this value in the user values ConfigMap to force the component to be restarted with the provided certificate. The value of the property should be the namespace and secret name which holds the certificate content.
+
+```yaml
+data:
+  values: |
+    controller:
+       extraArgs:
+         default-ssl-certificate: <namespace>/<secret_name>
+```
+
+### Custom annotation prefix
+
+By default we use the standard annotation prefix `nginx.ingress.kubernetes.io` in the ingress controller. In case you want to have a specific one this can be achieved via the user values ConfigMap:
+
+```yaml
+data:
+  values: |
+    controller:
+      extraArgs:
+        annotations-prefix: custom.prefix.io
+```
+
+### Web Application Firewall
+
+ingress nginx controller ships with the [ModSecurity](https://www.modsecurity.org) module which can be used for enhancing your ingress nginx controller deployment by Web Application Firewall capabilities to protect your workload against malicious requests.
+
+While enabling those capabilities in the first step is quite easy, it might take some more effort to actually fine-tune it to your needs. Since especially in blocking mode even legal requests might get blocked, we recommend to first run ModSecurity in the detection mode and observe its logs while it already checks incoming traffic for malicious requests.
+
+The included configuration is split into two parts: One is used for the basic setup of the ModSecurity module, the other is the [OWASP ModSecurity Core Rule Set](https://coreruleset.org) which provides a collection of rules against common and well-known attack vectors. While both can be enabled in parallel, the former will keep the whole ModSecurity in the before mentioned detection mode by default until explicitly configured otherwise:
```

Comment:
```suggestion
### Web Application Firewall

The ingress nginx controller ships with the [ModSecurity](https://www.modsecurity.org) module, which can be used to enhance your ingress nginx controller deployment by Web Application Firewall capabilities to protect your workload against malicious requests.

While enabling those capabilities in the first step is quite easy, it might take some more effort to fine-tune it to your needs. Since, especially in blocking mode, even legal requests might get blocked; we recommend first running ModSecurity in the detection mode and observing its logs. At the same time, it already checks incoming traffic for malicious requests.

The included configuration is split into two parts: One is used for the basic setup of the ModSecurity module, and the other is the [OWASP ModSecurity Core Rule Set](https://coreruleset.org), which provides a collection of rules against common and well-known attack vectors. While both can be enabled in parallel, the former will keep the whole ModSecurity in the detection above mode by default until explicitly configured otherwise:
```